mmv1/products/networksecurity/ServerTlsPolicy.yaml

```diff
@@ -47,26 +46,22 @@ import_format:
   ['projects/{{project}}/locations/{{location}}/serverTlsPolicies/{{name}}']
 examples:
   - !ruby/object:Provider::Terraform::Examples
-    min_version: beta
     name: 'network_security_server_tls_policy_basic'
     skip_vcr: true
```

Comment:
This PR fixes https://github.com/hashicorp/terraform-provider-google/issues/18103 so it doesn't need to have skip_vcr any more.

```suggestion
```